src/pybind/mgr/dashboard/frontend/src/app/shared/api/host.service.ts

```diff
@@ -21,6 +21,8 @@ export class HostService extends ApiClient {
   baseURL = 'api/host';
   baseUIURL = 'ui-api/host';
 
+  predefinedLabels = ['mon', 'mgr', 'osd', 'mds', 'rgw', 'nfs', 'iSCSI', 'rbd', 'grafana'];
```

Comment:
Everything lowercase?
```suggestion
  predefinedLabels = ['mon', 'mgr', 'osd', 'mds', 'rgw', 'nfs', 'iscsi', 'rbd', 'grafana'];
```

Reply:
done! Thanks @epuertat